package.json

```diff
@@ -12,23 +12,25 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "autotrack": "^1.0.0",
-    "babel-core": "^6.11.4",
-    "babel-eslint": "^6.1.2",
-    "babel-loader": "^6.2.4",
-    "babel-polyfill": "^6.9.1",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.11.1",
-    "babel-preset-stage-0": "^6.5.0",
-    "camelcase": "^3.0.0",
-    "core-js": "^2.4.1",
+    "babel-core": "^6.1.13",
```

Comment:
It looks like a lot of these have gone backwards. Can you make sure to include the latest changes from master?